test/statement_test.ts

```diff
@@ -30,9 +30,9 @@ describe('statements', () => {
   });
   it('translates try/catch', () => {
     expectTranslate('try {} catch(e) {} finally {}')
-        .to.equal(' try { } catch ( e ) { } finally { }');
+        .to.equal(' try { } catch ( e , e_stack ) { } finally { }');
```

Comment:
@mhevery, I believe it is only expensive when an error actually happens. During normal operation, errors should be very rare, so it should be ok to ask for stack trace for every error. @floitschG, could you please confirm this assumption?